design/prototype-version-4/edit-question/README.md

```diff
@@ -0,0 +1,91 @@
+
+A second secondary heading, “Question short name (optional)”, is followed by hint text “The short name will be used when the form’s questions are all displayed in a list. Use a short descriptive name. For example ‘Address’.”. This is followed by a text input.
+
+Below these is a closed details component showing a blue link with an arrow before the text “Add hint text to help people answer the question”.
+
+Next is a secondary heading, “What kind of answer do you need to this question?”, which has the hint text “The answer will be validated to check it’s in the selected format.” Below are radio buttons that determine the input type required:
+
+- Single line of text, selected
+- Address
+- Date
+- Email address
+- National Insurance number
+- Phone number
+
+The page ends with a green “Save question” button next to a grey “Create next question” button. Below the two buttons is a “Go to form overview” link.
+
+<!-- describe side preview pane -->
+On the right side of the screen there is a secondary heading, “Question preview” with a link to “Preview question in a new tab”.
+
+Below the link is a smaller version of an empty GOV.UK service page within an iframe, to mimic a mobile screen. It shows the GOV.UK logo on a black header. Within the body of the page is a disabled green ‘Continue’ button.
+
+
+### What we saw
+
+- User was unsure whether the save button would move them to another page
+- ‘Preview question in a new tab’ caused a navigation challenge when user tried to get out of the preview and back to form editing  
+- Some uncertainty around whether work had been saved when creating a question  
+- Some uncertainty as to what the preview pane did, particularly before users had saved a question and it had been populated with their content  
+- Users often needed to consider answer type before they were able to write useful hint text, but the current interface asks for hint text first  
+- They are also not shown how the answer type will display in the preview pane until they have saved a question. Some users re-wrote their hint text after saving and previewing  
+
+## What we changed and why
+
+- Removed the ‘short version’ input for a question. Users continued to query the need for it, and were unsure when or how they could use it. We decided that this is probably not needed as a basic feature, and if we return to consider it in the future it may be a more ‘advanced’ feature
+- Made the ‘hint text’ question and input always visible - no longer in a ‘details’ component - meaning it is more prominent
+- Added more context to the ‘save’ buttons making it clearer what action each will take
+- Made ‘save and add next question’ the main call to action when adding a new question to the end position in the form
+- Made ‘save and edit next question’ the main call to action when editing a question within the form journey, not last
```

Comment:
Not totally clear about the 'not last' bit at the end of line 57: does this mean that when editing a question within the form journey and that question is NOT the last question, 'save and edit next question' becomes the main call to action? 

This suggests to me that when editing a question that IS the last question, something else will be the main call to action - 'Save and preview question' perhaps? But is that right?

Reply:
It’s the latter - the bullet above is what the button (call to action) would be IF the current question the user is editing is the last one in their form.

Reply:
So am I right that in this prototype version we now have the options of:

- 'Save and add next question' as the main CtA when a new question is added in the end position
- 'Save and edit next question' as the main CtA when editing a question that is NOT in the end position
- 'Save and preview' as the main CtA when editing a question that IS the last question? 

Whereas before we had 'Create new question' or 'Save question' as the main CtAs, respectively?

Reply:
So in the version (v4) the only green/main CtAs are either: 

- 'Save and add next question' as the main CtA when a new question is added in the end position
- 'Save and edit next question' as the main CtA when editing a question that is NOT in the end position

The ‘Save and preview question’ is always the same secondary (grey) button

Reply:
Gottit :-)